src/OpenTelemetry.Instrumentation.EntityFrameworkCore/Implementation/EntityFrameworkDiagnosticListener.cs

```diff
@@ -124,12 +126,20 @@ public override void OnCustom(string name, Activity activity, object payload)
                         }
 
                         var dataSource = (string)this.dataSourceFetcher.Fetch(connection);
-
-                        activity.AddTag(AttributeDbName, database);
+                        activity.AddTag(AttributeDbName, activity.DisplayName);
```

Comment:
Could you please revert? I think that the previous version is more redable/obvious.